closed/custom/modules/java.base/Copy-post.gmk

```diff
@@ -52,6 +52,10 @@ $(call openj9_copy_files,, \
 	$(INCLUDE31_TARGET_DIR)/jniport.h)
 endif # zos
 
+$(call openj9_copy_files,, \
+	$(OPENJ9OMR_TOPDIR)/gc/verbose/schema.xsd \
+	$(LIB_DST_DIR)/$(OPENJ9_LIBS_SUBDIR)/schema.xsd)
```

Comment:
I think it should go up one directory level (i.e. remove `/$(OPENJ9_LIBS_SUBDIR)`).

Reply:
Made the change.